lib/core/Axios.js

```diff
@@ -66,6 +66,9 @@ Axios.prototype.request = function request(config) {
 
 Axios.prototype.getUri = function getUri(config) {
   config = mergeConfig(this.defaults, config);
+  if (!config.url) {
+    throw new Error('url of ' + config.url + 'is invalid for axios request');
```

Comment:
If the `config.url` is falsey (such an empty string), wouldn't this error message be not indicative?
in this example, if the url is `''` the error message would be `'url of is invalid for axios request'`